homework.md

```diff
@@ -44,70 +44,73 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+JavaScript is a scripting or programming language that allows you to implement complex features on web pages 
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+Git tracks the changes you make to files, so you have a record of what has been done.
+Can revert to specific versions should you ever need to. 
+Git also makes collaboration easier, allowing changes by multiple people to all be merged into one source.
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+A fork is a copy of a repository. Forking a repository allows you to freely experiment with changes without affecting the original project.
 
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+Cloning a repository pulls down a full copy of all the repository data that GitHub.com has at that point in time, including all versions of every file and folder for the project.
```

Comment:
Hi Mandeep. I would recommend checking forking vs cloning a repo. Your answer is good, but I think you could benefit from stating a clear difference between these two actions.